ee/clickhouse/models/property.py

```diff
@@ -85,19 +85,18 @@ def prop_filter_json_extract(
     prop: Property, idx: int, prepend: str = "", prop_var: str = "properties", allow_denormalized_props: bool = False
 ) -> Tuple[str, Dict[str, Any]]:
     # Once all queries are migrated over we can get rid of allow_denormalized_props
-    is_denormalized = prop.key.lower() in settings.CLICKHOUSE_DENORMALIZED_PROPERTIES and allow_denormalized_props
     json_extract = "trim(BOTH '\"' FROM JSONExtractRaw({prop_var}, %(k{prepend}_{idx})s))".format(
         idx=idx, prepend=prepend, prop_var=prop_var
     )
-    denormalized = "properties_{}".format(prop.key.lower())
```

Comment:
What's happening to these columns or is it accounted for by migration 0015?

Reply:
That migration handles it :)